package.json

```diff
@@ -37,7 +37,7 @@
     "chalk": "2.1.0",
     "config": "1.16.0",
     "console.table": "0.10.0",
-    "inquirer": "0.10.0",
+    "inquirer": "0.12.0",
```

Comment:
README should be modified.  Using this version the CLI gives the impression that it should work fine in git bash where as it really doesn't.

Reply:
Evgeni will modify the README in master